korman/ui/ui_world.py

```diff
@@ -26,6 +29,47 @@ def poll(cls, context):
         return context.world and context.scene.render.engine == "PLASMA_GAME"
 
 
+class PlasmaGamePanel(AgeButtonsPanel, bpy.types.Panel):
+    bl_label = "Plasma Games"
+
+    def draw(self, context):
+        layout = self.layout
+        games = context.world.plasma_games
+        age = context.world.plasma_age
+
+        row = layout.row()
+        row.template_list("PlasmaGameList", "games", games, "games", games,
+                          "active_game_index", rows=2)
+        col = row.column(align=True)
+        col.operator("world.plasma_game_add", icon="ZOOMIN", text="")
+        col.operator("world.plasma_game_remove", icon="ZOOMOUT", text="")
+
+        # Game Properties
+        active_game_index = games.active_game_index
+        if active_game_index < len(games.games):
+            active_game = games.games[active_game_index]
+
+            layout.separator()
+            box = layout.box()
+
+            box.prop(active_game, "path", emboss=False)
+            box.prop(active_game, "version")
+            box.separator()
+
+            row = box.row(align=True)
+            op = row.operator("world.plasma_game_add", icon="FILE_FOLDER", text="Change Path")
+            op.filepath = active_game.path
+            op.game_index = active_game_index
+            row.operator_context = "EXEC_DEFAULT"
+            op = row.operator("export.plasma_age", icon="EXPORT")
+            op.filepath = str((Path(active_game.path) / "dat" / age.age_name).with_suffix(".age"))
```

Comment:
When age_name is blank, we end up with a nice export of dat.age in the game's root folder. :trollface: We should probably refuse to export and give a pop-up error instead.